src/me.rs

```diff
@@ -15,11 +15,87 @@ use plane::*;
 use FrameInvariants;
 use FrameState;
 
+#[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
+#[target_feature(enable = "avx2")]
+unsafe fn get_sad_avx2(
+  plane_org: &mut PlaneSlice, plane_ref: &mut PlaneSlice, blk_h: usize,
+  blk_w: usize
+) -> u32 {
+  #[cfg(target_arch = "x86")]
+  use std::arch::x86::*;
+  #[cfg(target_arch = "x86_64")]
+  use std::arch::x86_64::*;
+
+  let mut sum = 0 as u32;
+
+  {
```

Comment:
Why this block?

Reply:
The intent is to drop the borrows `slice_org/slice_ref` have on `plane_org/plane_ref`, for the `plane_*.y += blk_h` at the end.